admin_manual/maintenance/upgrade.rst

```diff
@@ -15,6 +15,9 @@ Before beginning an upgrade, please keep the following points in mind:
 - Large installations may take several hours to complete the upgrade.
 - Downgrading **is not supported** as it risks corrupting your data. If you want to revert to an older ownCloud version, make a new, fresh installation and then restore your data from backup. Before doing this, file a support ticket (if you have paid support) or ask for help in the ownCloud forums to resolve your issue without downgrading.
 
+.. warning::
+   3rd Party Apps have to be disabled before the upgrade.
```

Comment:
3rd Party app = non core app?

I believe this is the explanation I am always given when I ask what a 3rd party app is.

I wanted to document this warning because in central there are many users who install ownCloud on a hosted server where they only have FTP access. Then they try to update with the web updater and then they get stuck. They would have to log in via ssh and perform the upgrade, then get the error message that the upgrade can't be performed unless they disable app 1, 2, and 3. Which at this point they only can do via command line.

If they don't have access to command line they are screwed. My thinking was, ( maybe I am wrong ) that if they disable the "non core" apps before the update with the web updater, the update would go through and they would not get stuck. That's why I wanted the warning here.

Either that or we make ssh access mandatory for using ownCloud and put a warning with this message somewhere in the doc.

Reply:
> we make ssh access mandatory for using ownCloud and put a warning with this message somewhere in the doc.

this is anyhow the way to go from my pov.

As explained above - disabling all apps by default can cause more issues then solving.
This is also the reason why we don't auto disable all apps which cannot be updated.

I was explaining this somewhere on central or in an core issue these days.